apps/backend/src/infra/mongo/schemas/user-session.schema.ts

```diff
@@ -2,6 +2,38 @@ import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
 import { HydratedDocument, SchemaTimestampsConfig, Types } from 'mongoose';
 import { MongoUser } from './user.schema';
 
+function parseDurationToSeconds(duration: string): number {
```

Comment:
I wonder if this isn't a good opportunity to install [dayjs](https://day.js.org/) and use it for future operations with dates. This library is the one recommended for react native